packages/addons-v5/test/commands/addons.--all.js

```diff
@@ -27,11 +27,11 @@ describe('addons --all', function () {
     it('prints add-ons in a table', function () {
       return cmd.run({ flags: {} }).then(function () {
         util.expectOutput(cli.stdout,
-          `Owning App    Add-on     Plan                         Price      State
-────────────  ─────────  ───────────────────────────  ─────────  ────────
-acme-inc-api  api-redis  heroku-redis:premium-2       $60/month  created
-acme-inc-www  www-db     heroku-postgresql:hobby-dev  free       created
-acme-inc-www  www-redis  heroku-redis:premium-2       $60/month  creating`)
+          `Owning App    Add-on     Plan                       Price      State
+────────────  ─────────  ─────────────────────────  ─────────  ────────
+acme-inc-api  api-redis  heroku-redis:premium-2     $60/month  created
+acme-inc-www  www-db     heroku-postgresql:no-free  free       created
```

Comment:
The price is showing up as `free` in the output, but since we're removing free stuff in here, should that be the case?